app/join-us/page.tsx

```diff
@@ -174,8 +182,11 @@ export default function JoinUsPage() {
       reason: reason.trim(),
       otherClubs: otherClubs.trim() || "None",
       preferredTeam: preferredTeams.join(", "),
+      token,
     }
 
+    console.log("reCAPTCHA token:", token)
```

Comment:
Better to log only in development, because the token is valid for 2 minutes, that's long enough to allow a replay attack."
```suggestion
  if (process.env.NODE_ENV === "development") {
    console.log("reCAPTCHA token:", token)
  }
```

Reply:
Thanks for pointing that out! I’ve updated the log to run only in development mode using process.env.NODE_ENV === "development", preventing token exposure in production.